### PR TITLE
chore: Unit test modules removed

### PR DESCRIPTION
**Pull request by hendrikebbers**
This PR moves tests and testFixtures from the deprecated modules at `platform-sdk/swirlds-unit-tests` to the correct modules.
Part of https://github.com/hashgraph/hedera-services/issues/5293

### CODE REVIEW

File: platform-sdk/swirlds-jasperdb/src/test/java/com/swirlds/merkledb/files/MockAppender.java [hunk 1]
```diff
@@ -14,9 +14,8 @@
  * limitations under the License.
  */
 
-package com.swirlds.common.test.logging;
+package com.swirlds.merkledb.files;
```
Comment (mxtartaglia-sl) [inline platform-sdk/swirlds-jasperdb/src/test/java/com/swirlds/merkledb/files/MockAppender.java:17]: Is this package correct? it seems unrelated to merkledb

**Review by mxtartaglia-sl — COMMENTED** [review]
@hendrikebbers Just a question, i am prepared to approve in any case.

File: platform-sdk/swirlds-jasperdb/src/test/java/com/swirlds/merkledb/files/MockAppender.java (hunk 1 above)
Comment (hendrikebbers) [reply platform-sdk/swirlds-jasperdb/src/test/java/com/swirlds/merkledb/files/MockAppender.java:17]: I assume it is only used by merkedb tests.

**Review by mxtartaglia-sl — APPROVED** [review]

**Review by jeromy-cannon — APPROVED** [review]

File: platform-sdk/swirlds-jasperdb/src/test/java/com/swirlds/merkledb/files/MockAppender.java [hunk 2]
```diff
@@ -38,15 +37,15 @@
...
+    public MockAppender(final String name) {
         super("MockAppender-" + name, null, null, true, null);
     }
 
     /**
      * {@inheritDoc}
      */
     @Override
-    public void append(@NonNull final LogEvent event) {
+    public void append(final LogEvent event) {
```
Comment (anthony-swirldslabs) [inline platform-sdk/swirlds-jasperdb/src/test/java/com/swirlds/merkledb/files/MockAppender.java:48]: Minor: while `@NonNull` may not be my personal favorite annotation, it's still unclear why this fix removes them from here. From the code logic perspective, they kind of make sense here. And this change doesn't seem to be related to the "unit test modules removal". So I'm not sure what this diff is doing here.

File: platform-sdk/swirlds-config-extensions/src/testFixtures/java/com/swirlds/config/extensions/test/fixtures/TestConfigBuilder.java [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2023-2024 Hedera Hashgraph, LLC
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
```
Comment (jsync-swirlds) [inline platform-sdk/swirlds-config-extensions/src/testFixtures/java/com/swirlds/config/extensions/test/fixtures/TestConfigBuilder.java:2]: Are we certain we should be removing the earlier years on these?

**Review by anthony-swirldslabs — APPROVED** [review]

**Review by jsync-swirlds — COMMENTED** [review]
One small question about the (many) changes to "copyright" headers.

File: platform-sdk/swirlds-config-extensions/src/testFixtures/java/com/swirlds/config/extensions/test/fixtures/TestConfigBuilder.java (hunk 3 above)
Comment (hendrikebbers) [reply platform-sdk/swirlds-config-extensions/src/testFixtures/java/com/swirlds/config/extensions/test/fixtures/TestConfigBuilder.java:2]: that has been created by (I assume) spotless

Comment (hendrikebbers) [reply platform-sdk/swirlds-config-extensions/src/testFixtures/java/com/swirlds/config/extensions/test/fixtures/TestConfigBuilder.java:2]: @nathanklick what is the wished format?

**Review by cody-littley — DISMISSED** [review]

**Review by Neeharika-Sompalli — APPROVED** [review]
LGTM  from `hedera-base` changes.

**Comment by jsync-swirlds** [comment]
compile fails:

platform-sdk/swirlds-unit-tests/core/swirlds-platform-test/src/test/java/com/swirlds/platform/test/sync/SyncFilteringTest.java:35: error: package com.swirlds.test.framework.context does not exist

**Review by jeromy-cannon — APPROVED** [review]

**Review by anthony-swirldslabs — DISMISSED** [review]

**Comment by hendrikebbers** [comment]
@jsync-swirlds yeah, just clicking "update branch" in GitHub killed it ;) Will refactor it today...

**Review by jeromy-cannon — DISMISSED** [review]

**Review by mxtartaglia-sl — DISMISSED** [review]

File: README.md [hunk 4]
```diff
@@ -8,7 +8,7 @@
 [![Development Branch](https://img.shields.io/badge/docs-quickstart-green.svg)](hedera-node/docs/gradle-quickstart.md)
 [![License](https://img.shields.io/badge/license-apache2-blue.svg)](LICENSE)
 
-# Hedera Services
+# Hedera Services TEST TEST
```
Comment (jsync-swirlds) [inline README.md:11]: leftover?

**Review by jsync-swirlds — DISMISSED** [review]

**Review by jeromy-cannon — DISMISSED** [review]

**Review by lpetrovic05 — DISMISSED** [review]

**Review by mxtartaglia-sl — DISMISSED** [review]

**Review by kimbor — DISMISSED** [review]
lgtm!

**Review by anthony-swirldslabs — APPROVED** [review]

**Review by cody-littley — DISMISSED** [review]

**Review by jeromy-cannon — APPROVED** [review]

**Review by anthony-swirldslabs — APPROVED** [review]

**Review by jeromy-cannon — APPROVED** [review]

**Review by mxtartaglia-sl — APPROVED** [review]

**Review by mxtartaglia-sl — DISMISSED** [review]

**Review by jeromy-cannon — DISMISSED** [review]

**Review by mxtartaglia-sl — APPROVED** [review]

**Review by jsync-swirlds — APPROVED** [review]

**Review by edward-swirldslabs — APPROVED** [review]

**Review by jeromy-cannon — APPROVED** [review]

**Review by lpetrovic05 — APPROVED** [review]

**Review by timo0 — APPROVED** [review]
